rpm/libksba.spec

```diff
@@ -1,6 +1,6 @@
 Name:       libksba
 Summary:    X.509 library
-Version:    1.3.5
+Version:    1.6.2
 Release:    1
 License:    GPLv2+ or LGPLv3+
 URL:        https://github.com/sailfishos/libksba
```

Comment:
Is the texinfo required here? Can this be built without the info docs? Can we avoid this https://github.com/sailfishos-mirror/libksba/blob/master/Makefile.am#L45 ?

Reply:
Disabled info documentation using configure option.